docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -0,0 +1,40 @@
+---
+id: nexus
+title: Temporal Nexus - Temporal feature
+description: Explore how Temporal Nexus enables durable execution across team and namespace boundaries and promotes a modular architecture that enables each team to have their own namespace for improved security, troubleshooting, and blast radius isolation.
+sidebar_label: Temporal Nexus
+slug: /nexus
+tags:
+  - temporal-nexus
+  - temporal-cloud
+
+keywords:
+  - temporal multi-team collaboration
+  - modular temporal architecture
+  - namespace boundary workflows
+  - temporal nexus api registry
+  - temporal nexus services
+  - secure temporal workflows
+  - multi-team integration temporal
+  - temporal cloud nexus
+---
+
+Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries - available in Public Preview for OSS and Temporal Cloud. 
```

Comment:
```suggestion
Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries - available in Public Preview for self-hosted deployments and Temporal Cloud. 
```

I would also add an asterisk on "cloud boundaries" and "self-hosted" to mention:
- Cross cloud calls are not available yet on Temporal Cloud
- Only single cluster deployments are supported for self-hosted